public/ably.php

```diff
@@ -0,0 +1,5 @@
+<?php
+require_once __DIR__ . '/../vendor/autoload.php';
+
+throw new \RuntimeException('Insert your API key'); // REMOVE THIS
+$ably = new \Ably\AblyRest(["key" => "{{ApiKey}}"]);
```

Comment:
tutorial for ruby and node used similar syntax, and in docs i've copied that block from "history" tutorial, i'll update it to be correct with code sample